From 66b0b0b05cc49bf0a15aa85204775d83aa99078a Mon Sep 17 00:00:00 2001
From: Andrew Bayer <andrew@cloudera.com>
Date: Mon, 12 Mar 2012 15:31:21 -0700
Subject: [PATCH 0004/1357] CLODUERA-BUILD. Make modules/*.jar in tarball have proper versions.

This fixes the jar names in the modules directory of the dist when
they're SNAPSHOTs, by forcing the use of project.version in their name,
rather than module.version - module.version ends up with the unique
snapshot identifier, which causes a lot of problems.
---
 .../resources/assemblies/hadoop-mapreduce-dist.xml |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/hadoop-assemblies/src/main/resources/assemblies/hadoop-mapreduce-dist.xml b/hadoop-assemblies/src/main/resources/assemblies/hadoop-mapreduce-dist.xml
index fff40cb..ce0217f 100644
--- a/hadoop-assemblies/src/main/resources/assemblies/hadoop-mapreduce-dist.xml
+++ b/hadoop-assemblies/src/main/resources/assemblies/hadoop-mapreduce-dist.xml
@@ -133,6 +133,7 @@
         <outputDirectory>share/hadoop/${hadoop.component}</outputDirectory>
         <includeDependencies>false</includeDependencies>
         <unpack>false</unpack>
+        <outputFileNameMapping>${module.artifactId}-${project.version}${dashClassifier?}.${module.extension}</outputFileNameMapping>
       </binaries>
     </moduleSet>
     <moduleSet>
-- 
1.7.0.4

